src/OpenProject.Shared/ConfigurationConstant.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.IO;
+
+namespace OpenProject.Shared
+{
+  public static class ConfigurationConstant
+  {
+    public static string OpenProjectApplicationData =
+      Path.Combine(
+        Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
+        "OpenProject.Revit");
```

Comment:
I wonder if we maybe should place this in a constant somewhere. This should maybe be synchronized with the product name from the installer, and I think this being just a string literal here makes it very easy in the future to miss it when things change.

Reply:
Usually I think it is enough, if a constant value like this is defined only at one location. But I also see now, I missed a `readonly` here.

When you say, making it a constant, what do you suggest? Putting it to a config file loaded at runtime? a `const` is at least not possible, when we want to put it into the `AppData` special folder, as there is no constant for that and is has to be evaluated.

Reply:
Sorry for the confusion, with `const` I really only meant the `OpenProject.Revit` string literal.

The value itself is fine, but I was imagining this scenario: If a new developer comes along at some later point, and they want to look at the logs on their machine, where do they find them? Will this maybe ever change in the future, if we decide to rename the plugin? Also, what if we change the plugin name to something else?

So I was more thinking about if we could somehow connect this folder name to the product name of the app. It's probably not really relevant, and maybe never will, just something I noticed. I don't think any action is required.